ChangeFeedLabSolution/Startup.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Azure.Messaging.EventHubs.Producer;
+using Microsoft.Azure.Functions.Extensions.DependencyInjection;
+using Microsoft.Extensions.Azure;
+
+[assembly: FunctionsStartup(typeof(ChangeFeedFunction.Startup))]
+namespace ChangeFeedFunction
+{
+    public class Startup : FunctionsStartup
+    {
+        public const string EventHubName = "event-hub1";
+        public const string EventHubNamespaceConnection = "<< CONNECTION STRING FOR THE EVENT HUBS NAMESPACE >> ";
+        
+        public override void Configure(IFunctionsHostBuilder builder)
+        {
+            builder.Services.AddAzureClients(builder =>
+            {
+                var producer = new EventHubProducerClient(EventHubNamespaceConnection, EventHubName);
```

Comment:
why are we creating a producer here and then doing nothing with it?   I think we can simplify this to just use the `AddEventHubProducerClient` call.

Reply:
@jsquire - Removed.